File: Wrap/Wrap java/Wrap4.java
import java.util.Scanner;
public class Wrap4{

    public static void main(String[] args){

    Scanner input = new Scanner(System.in);

    System.out.println("Enter a number");
    int number = input.nextInt();
    
    int sum = 0;

    for(int count = 1; count < number; count++){ 
       
        if(number % count == 0){

            sum += count;         
        }
 
    }

    if(sum == number){
            System.out.println("The number is a perfect number");
    }
    else{
            System.out.println("The number is not a perfect number"); 
    }
       
    }

}

File: Wrap/ Wrap JavaScript/Wrap4.js
const scanner = require("prompt-sync")();
const number = scanner("Enter a number:");
    
    let sum = 0;

    for(let count = 1; count < number; count++){ 
       
        if(number % count == 0){

            sum += count;         
        }
 
    }

    if(sum == number){
            console.log("The number is a perfect number");
    }
    else{
            console.log("The number is not a perfect number");

    }
